packages/blueprints/blueprint/src/blueprint.ts

```diff
@@ -142,12 +142,18 @@ export class Blueprint extends Project {
      * copy all non-src file from proposedBundle into the resolved bundle
      * only src is merge constructed.
      */
-    const supersetNonSourcePaths: string[] = filepathSet([proposedBundle], ['**/*', '!src/**']);
-    for (const filepath of supersetNonSourcePaths) {
-      const outputPath = path.join(this.outdir, filepath);
-      fs.mkdirSync(path.dirname(outputPath), { recursive: true });
-      const filecontent = fs.readFileSync(path.join(proposedBundle, filepath));
-      fs.writeFileSync(outputPath, filecontent);
+    const supersetNonSourceNonIssuesPaths: string[] = filepathSet([proposedBundle], ['**/*', '!src/**', '!issues/**']);
+    for (const filepath of supersetNonSourceNonIssuesPaths) {
+      this.writeNonSourceFile(filepath, proposedBundle);
+    }
+
+    /**
+     * copy all issue src files that are unique to the proposedBundle into the resolved bundle
+     * to prevent duplicate issue creation
+     */
+    const setUniqueToProposedBundle = filepathDiffBetweenSets_B_A(ancestorBundle, proposedBundle, ['issues/**']);
```

Comment:
This diff should be handled by the issue component

Reply:
resolved through discussion